src/Config/Auth.php

```diff
@@ -325,4 +325,30 @@ class Auth extends BaseConfig
      * @var class-string<\Sparks\Shield\Interfaces\UserProvider>
      */
     public $userProvider = 'Sparks\Shield\Models\UserModel';
+
+    /**
+     * Returns the URL that a user should be redirected
+     * to after a successful login.
+     */
+    public static function loginRedirect(): string
```

Comment:
These will work on instances but I'm worried having them static might encourage developers to use them statically - any reason not to drop the static altogether?
```suggestion
    public function loginRedirect(): string
```